sway-core/src/type_system/ast_elements/type_argument.rs

```diff
@@ -92,23 +96,19 @@ impl GenericArgument {
             .initial_type_id
     }
 
-    pub fn call_path_tree(&self) -> Option<&CallPathTree> {
-        match self {
-            GenericArgument::Type(a) => a.call_path_tree.as_ref(),
-            GenericArgument::Const(_) => {
-                todo!("Will be implemented by https://github.com/FuelLabs/sway/issues/6860")
-            }
-        }
-    }
+    // pub fn call_path_tree(&self) -> Option<&CallPathTree> {
```

Comment:
Do we want to keep this commented code? Looks like it stayed unintentionally.

Reply:
Removed for now. We may want to bring them back later.